prow/github/client.go

```diff
@@ -386,6 +386,24 @@ func (c *Client) CreateStatus(org, repo, ref string, s Status) error {
 	return err
 }
 
+func (c *Client) GetLabels(org, repo string) ([]string, error) {
```

Comment:
suggestions for a way of doing this? do i just have to increment the page option and keep getting the labels until nothing is returned? sounds pretty terrible